apps/protocol-frontend/src/components/BulkDaoAttributeModal.tsx

```diff
@@ -0,0 +1,94 @@
+import { useState, useCallback } from 'react';
+import { Select } from '@govrn/protocol-ui';
+import { Stack, Button, Text, Progress } from '@chakra-ui/react';
+import { useForm, SubmitHandler } from 'react-hook-form';
+import { yupResolver } from '@hookform/resolvers/yup';
+import { useUser } from '../contexts/UserContext';
+import { editContributionFormValidation } from '../utils/validations';
+import { BulkDaoAttributeFormValues } from '../types/forms';
+
+interface BulkDaoAttributeModalProps {
+  contributions: any;
```

Comment:
Would merging in Amrros pr resolve these

Reply:
Yup, I think that if we merge in Amrro's PR first we should be able to then fill in all the Contribution, Contribution[], Attestation, and Attestation[] types

If we do that first, then I think we'll be in better shape with types except for the ones stemming from React Table v7